pkgs/tools/misc/fsrx/default.nix

```diff
@@ -2,16 +2,16 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "fsrx";
-  version = "1.0.0";
+  version = "1.0.2";
 
   src = fetchFromGitHub {
     owner = "coloradocolby";
```

Comment:
The repo was moved to https://github.com/thatvegandev/fsrx, please also update this and meta.homepage

Reply:
Oh, I didn't even notice.

Reply:
Thank you =)